Answers.md

```diff
@@ -1,5 +1,19 @@
 1.  Explain the differences between `client-side routing` and `server-side routing`.
-1.  What does HTTP stand for?
-1.  What does CRUD stand for?
-1.  Which HTTP methods can be mapped to the CRUD acronym that we use when interfacing with APIs/Servers.
-1.  Mention three tools we can use to make AJAX requests
\ No newline at end of file
+
+During Client-side routing, the whole website or web-application needs to be loaded on the first request (That’s why the initial loading time usually takes longer), mean while, the server-side route will only request the data that’s needed.
```

Comment:
Review the difference between client-side routing and server-side routing.

Remember that client-side only needs to load from the server once.